project1/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Nunito+Sans|Playfair+Display|Titillium+Web&display=swap"
+        rel="stylesheet">
+    <title>project1_The Crane Wife</title>
+</head>
+
+
+
+<body>
+    <header>
+
+        <h1>The Crane Wife</h1>
+        <h2><cite>--by CJ Hauser</h2></cite>
+        <h3>July 16, 2019</h3>
+    </header>
+    <p>Ten days after I called off my engagement I was supposed to go on a scientific expedition to study the whooping
+        crane on the gulf coast of Texas. <span class="thought"><i>Surely, I will cancel this trip</i></span>, I
+        thought, as I shopped for nylon
+        hiking pants that zipped off at the knee. <span class="thought"><i>Surely, a person who calls off a wedding is
+                meant to be sitting
+                sadly at home, reflecting on the enormity of what has transpired and not doing whatever it is I am about
+                to
+                be doing that requires a pair of plastic clogs with drainage holes. Surely</i></span>, I thought, as I
+        tried on a
+        very large and floppy hat featuring a pull cord that fastened beneath my chin, it would be wrong to even be
+        wearing a hat that looks like this when something in my life has gone so <em>terribly wrong.</em></p>
+
+    <p> Ten days earlier I had cried and I had yelled and I had packed up my dog and driven away from the upstate
+        New York house with two willow trees I had bought with my fiancé.</p>
+
+    <p><em>Ten days later and I didn’t want to do anything I was supposed to do.</em></p>
+
+
+    <p>I went to Texas to study the <i>whooping crane</i> because I was researching a novel. In my novel there were
```

Comment:
I would reserve the `i` element for the scientific name of the crane, not it's common name. See [Wikipedia's example](https://en.wikipedia.org/wiki/Whooping_crane).